practicals/LLM_prac.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
<p>A few susgestions:</p><p> </p><ol><li>Given that most people attending are beginners can we have the Attention Mechanism section labeled as <code>Beginner</code> and target a very fundamental overview / breakdown of attention?</li><li>Change RLHF in content to "RLHF, Multimodal Transformers, and other relevant developments related to LLMs"</li><li>Rename: "Building a decoder only transformer from scratch" -&gt; "Understand and implement the fundamental building blocks of the Transformer Architecture"</li><li>Remove the "Understanding the RLHF training method" objective; we could instead say that "Introduce references to trending research and applications of LLMs" e.g. RLHF, MOE, Multimodal and multilingual transformers, sparsity and whatever else we could gather, for this section we can provide a very light definition/introduction and point to existing resources for more</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>